apps/myaccount/src/components/account-recovery/account-recovery.tsx

```diff
@@ -45,12 +45,13 @@ export const AccountRecoveryComponent: React.FunctionComponent<AccountRecoveryPr
     props: AccountRecoveryProps
 ): JSX.Element => {
     const { t } = useTranslation();
-    const { onAlertFired, featureConfig } = props;
+    const { onAlertFired, featureConfig, ['data-testid']: testId } = props;
```

Comment:
Shall we reformat this as follows. Please check other places as well.

```suggestion
    const {
        onAlertFired,
        featureConfig,
        ['data-testid']: testId
    } = props;
```

Reply:
Reformatted as required. However, we can keep object de-structuring statements inline if it's has only like 2 properties right? if property count is 3 or more let's reformat to multiple lines.